go/service/kbfs_mount.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package service
+
+import (
+	"golang.org/x/net/context"
+	"os"
+
+	"github.com/keybase/client/go/libkb"
+	//	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+)
+
+type KBFSMountHandler struct {
+	*BaseHandler
+	libkb.Contextified
+}
+
+func NewKBFSMountHandler(xp rpc.Transporter, g *libkb.GlobalContext) *KBFSMountHandler {
+	return &KBFSMountHandler{
+		BaseHandler:  NewBaseHandler(xp),
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+func (h *KBFSMountHandler) GetCurrentDriveLetter(ctx context.Context) (res string, err error) {
+
+	mountdir := os.Getenv("KEYBASE_DRIVE_LETTER")
+	if mountdir == "" {
+		i, err := h.G().Env.GetConfig().GetInterfaceAtPath("mountdir")
```

Comment:
You shouldn't need to use `GetInterfaceAtPath` --- can `GetStringAtPath` work here?